nss-day-cohort-43/nuttyshell-the-coven-code
# Rendered header with JS

# Description

- The header in the authorization screen is no longer hard-coded. I put all header components together in folder structure.

- I also fixed a bug that I found along the way: the logout button was lost if page was refreshed.

## Type of change


# Testing Instructions
Try refreshing after logging in and after registering to confirm that logout button is still there. While doing that, add in a logout or two to confirm that authHeader is visible when you need it.
# Checklist:
